slack/web/base_client.py

```diff
@@ -274,19 +274,6 @@ def _urllib_api_call(
         files: Dict[str, io.BytesIO] = {},
         additional_headers: Dict[str, str] = {},
     ) -> SlackResponse:
-        """Performs a Slack API request and returns the result.
```

Comment:
The `slack` package exists just for v2 backward compatibility (we will remove these in the long run). To reduce duplicated work for the future, let me delete these comments.